app/controllers/api/v2/courses_controller.rb

```diff
@@ -56,6 +56,27 @@ def update
         enrichment.assign_attributes(update_params)
         enrichment.save
 
+        # todo: something more canonical
+        # todo: validate sites actually belong to this org
+        sites = params["_jsonapi"]["data"]["relationships"]["sites"]
```

Comment:
Are we not able to use the deserializer here? As in, use `params['sites']`?

Reply:
Sorta; how about `params[:course][:sites_ids]`?